api/agent/agent.go

```diff
@@ -1115,21 +1087,21 @@ func (a *agent) runHotReq(ctx context.Context, call *call, state ContainerState,
 				state.UpdateState(ctx, ContainerStatePaused, call.slots)
 			}
 			continue
-		case <-evictor.C:
+		case <-c.GetEvictChan():
```

Comment:
I don't think this can happen right now, since `SetEvictable` always creates a token.  However, should `GetEvictChan()` return `nil` because an `evictToken` is absent, it looks like this will block forever.

Reply:
I documented this as much as possible now, GetEvictChan() returned channel must not be held beyond Enable/DisableEviction() calls (since these can alloc/realloc the token.)